druid-shell/src/windows/win_main.rs

```diff
@@ -84,7 +84,7 @@ impl RunLoop {
 
                 // Handle windows messages
                 loop {
-                    let mut msg = mem::uninitialized();
+                    let mut msg = mem::MaybeUninit::<MSG>::zeroed().assume_init();
```

Comment:
@cmyr Am unable to find `MSG`, which is supposed to be the type for msg here.

`RECT` should be fine to zero, as it just contains four numerical values.
`DXGI_ADAPTER_DESC` is confusing to me. The type declaration is as follows:
```rust
struct DXGI_ADAPTER_DESC {
    Description: [WCHAR; 128],
    VendorId: UINT,
    DeviceId: UINT,
    SubSysId: UINT,
    Revision: UINT,
    DedicatedVideoMemory: SIZE_T,
    DedicatedSystemMemory: SIZE_T,
    SharedSystemMemory: SIZE_T,
    AdapterLuid: LUID,
}
```
My search-foo fails me here, am unable to find out what `LUID` is. On the other hand, I assume that Rust initializes the vector for `Description` within the struct instead of using a pointer to an external vector, which would make zeroing it safe.